packages/create-sitecore-jss/src/templates/angular/src/app/jss-graphql.module.ts

```diff
@@ -68,14 +65,9 @@ export class GraphQLModule {
     // ...or a batched link (multiple queries within 10ms all go in one HTTP request)
     const batchHttp = this.httpLink.create({
       uri: environment.graphQLEndpoint,
-      headers: sc_apikey
+      headers: sc_apikey,
     });
 
-    // ...and an automatic persisted query link, which reduces bandwidth by using query hashes to alias content
-    // the APQ link is _chained_ behind another link that performs the actual HTTP calls, so you can choose
-    // APQ + batched, or APQ + http links for example.
-    const automaticPersistHttp = createPersistedQueryLink({ sha256 }).concat(batchHttp);
```

Comment:
Can we leave this as a comment instead of deleting it? The limitation is only present when using experience edge correct? I would assume this will still have benefits to other customers?